### PR TITLE
CC-26804: Add an integration guide for product offer shipment type feature

### PR DESCRIPTION
**Pull request by ievgen-malykhin**
Ticket: https://spryker.atlassian.net/browse/CC-26804

## PR Description

Added an integration guide for `Product offer shipment type` feature

## Checklist
- [X] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: _includes/pbc/all/install-features/202304.0/install-the-product-offer-shipment-feature.md [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the following module has been installed:
+
+| MODULE       | EXPECTED DIRECTORY           |
+|--------------|------------------------------|
+| ProductOffer | vendor/spryker/product-offer |
+| Shipment     | vendor/spryker/shipment      |
```
Comment (abitskil) [inline _includes/pbc/all/install-features/202304.0/install-the-product-offer-shipment-feature.md:29]: * The connector module has to be in place (ProductOfferShipmentType).

**Review by abitskil — DISMISSED** [review]

**Review by andriispryker — APPROVED** [review]
Approved.

File: _includes/pbc/all/install-features/202304.0/install-the-product-offer-shipment-feature.md [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+### Prerequisites
+
+To start feature integration, integrate the required features:
+
+| NAME          | VERSION          | INTEGRATION GUIDE                                                                                                                                         |
+|---------------|------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+ | Product Offer | {{site.version}} | [Product Offer feature integration](/docs/marketplace/dev/feature-integration-guides/{{site.version}}/marketplace-product-offer-feature-integration.html) |
+ | Shipment      | {{site.version}} | [Shipment feature integration](/docs/pbc/all/carrier-management/{{page.version}}/install-and-upgrade/install-the-shipment-feature.html)                   |
+
+## 1) Install the required modules using Composer
```
Comment (AlexSlawinski) [inline _includes/pbc/all/install-features/202304.0/install-the-product-offer-shipment-feature.md:16]:
```suggestion
## Install the required modules using Composer
```

File: _includes/pbc/all/install-features/202304.0/install-the-product-offer-shipment-feature.md [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+
+| MODULE                   | EXPECTED DIRECTORY                         |
+|--------------------------|--------------------------------------------|
+| ProductOffer             | vendor/spryker/product-offer               |
+| Shipment                 | vendor/spryker/shipment                    |
+| ProductOfferShipmentType | vendor/spryker/product-offer-shipment-type |
+
+{% endinfo_block %}
+
+### 2) Set up behavior
```
Comment (AlexSlawinski) [inline _includes/pbc/all/install-features/202304.0/install-the-product-offer-shipment-feature.md:34]:
```suggestion
### Set up behavior
```

File: _includes/pbc/all/install-features/202304.0/install-the-product-offer-shipment-feature.md [hunk 4]
```diff
@@ -0,0 +1,263 @@
...
+
+Enable the following plugins.
```
Comment (AlexSlawinski) [inline _includes/pbc/all/install-features/202304.0/install-the-product-offer-shipment-feature.md:36]:
```suggestion
Enable the following plugins:
```

File: _includes/pbc/all/install-features/202304.0/install-the-product-offer-shipment-feature.md [hunk 5]
```diff
@@ -0,0 +1,263 @@
...
+     */
+    protected function getProductOfferExpanderPlugins(): array
+    {
+        return [
+            new ShipmentTypeProductOfferExpanderPlugin(),
+        ];
+    }
+```
+
+### 3) Set up database schema and transfer objects
```
Comment (AlexSlawinski) [inline _includes/pbc/all/install-features/202304.0/install-the-product-offer-shipment-feature.md:89]:
```suggestion
### Set up database schema and transfer objects
```

File: _includes/pbc/all/install-features/202304.0/install-the-product-offer-shipment-feature.md [hunk 6]
```diff
@@ -0,0 +1,263 @@
...
+| Country                        | class | created | src/Generated/Shared/Transfer/CountryTransfer                        |
+| StoreRelation                  | class | created | src/Generated/Shared/Transfer/StoreRelationTransfer                  |
+| Totals                         | class | created | src/Generated/Shared/Transfer/TotalsTransfer                         |
+| Filter                         | class | created | src/Generated/Shared/Transfer/FilterTransfer                         |
+| OrderFilter                    | class | created | src/Generated/Shared/Transfer/OrderFilterTransfer                    |
+| Sort                           | class | created | src/Generated/Shared/Transfer/SortTransfer                           |
+
+{% endinfo_block %}
+
+### 5) Import shipment types for product offers
```
Comment (AlexSlawinski) [inline _includes/pbc/all/install-features/202304.0/install-the-product-offer-shipment-feature.md:174]:
```suggestion
### Import shipment types for product offers
```

File: _includes/pbc/all/install-features/202304.0/install-the-product-offer-shipment-feature.md [hunk 7]
```diff
@@ -0,0 +1,263 @@
+This document describes how to integrate the Product offer shipment feature into a Spryker project.
+
+## Install feature core
+
+Follow the steps below to install the Product offer shipment feature core.
+
+### Prerequisites
```
Comment (AlexSlawinski) [inline _includes/pbc/all/install-features/202304.0/install-the-product-offer-shipment-feature.md:7]:
```suggestion
## Prerequisites
```

File: _includes/pbc/all/install-features/202304.0/install-the-product-offer-shipment-feature.md [hunk 8]
```diff
@@ -0,0 +1,263 @@
...
+
+To start feature integration, integrate the required features:
```
Comment (AlexSlawinski) [inline _includes/pbc/all/install-features/202304.0/install-the-product-offer-shipment-feature.md:9]:
```suggestion
To start feature integration, integrate the following required features:
```

**Pull request merged**